gnome-initial-setup/pages/summary/gis-summary-page.c

```diff
@@ -175,12 +175,38 @@ add_uid_file (uid_t uid)
 }
 
 static void
+reload_systemd_service (GDBusConnection *connection, const char *service)
+{
+  GError *error = NULL;
+  GVariant *rv = g_dbus_connection_call_sync (connection,
+                                              "org.freedesktop.systemd1",
+                                              "/org/freedesktop/systemd1",
+                                              "org.freedesktop.systemd1.Manager",
+                                              "ReloadUnit",
+                                              g_variant_new ("(ss)", service, "restart"),
+                                              G_VARIANT_TYPE ("(s)"),
+                                              G_DBUS_CALL_FLAGS_NONE,
+                                              G_MAXINT, NULL, &error);
+
+  if (!rv && error)
```

Comment:
You should not need to check for both `!rv` and `error` here -- just the error should be enough.